manifests/bosh-manifest/deployments/aws/090-bosh-aws-stub.yml

```diff
@@ -21,7 +21,6 @@ meta:
 
   bosh_private_ip: (( grab terraform_outputs.microbosh_static_private_ip ))
```

Comment:
We can probably remove `bosh_private_ip` and `bosh_public_ip` and use terraform_outputs directly when reserving the IPs.
`bosh_public_ip` is also used in `default_dns` but I don't think we use it.